reconcile/change_owners/changes.py

```diff
@@ -106,6 +95,13 @@ def cover_changes(self, change_type_context: ChangeTypeContext) -> list[Diff]:
         The change-type contexts that cover a change, are registered in the
         `DiffCoverage.coverage` list right next to the Diff that is covered.
         """
+
+        # if this is a metadata only change, register the change type context
+        # as a source of approvers
+        if self.metadata_only_change and not self.diffs:
+            self._metadata_only_diff_coverage().coverage.append(change_type_context)
+            return []
```

Comment:
this `return` means there is no coverage for this change, right ?

Reply:
the return value of this function is only used in tests to assert about what happened. i never liked this hack.
let me fix this.

Reply:
and it is gone